src/test/resources/pages/instructorSearchPageSearchComments.html

```diff
@@ -135,7 +135,7 @@
                   </b>
                   [Tue, 01 May 2012, 08:49 AM UTC]
                 </span>
-                <a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=searchUI.idOfInstructor1OfCourse1&courseid=searchUI.idOfTypicalCourse1#${comment.id}" style="display:none;" target="_blank" title="" type="button">
+                <a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=searchUI.idOfInstructor1OfCourse1&courseid=searchUI.idOfTypicalCourse1#${comment.id}" style="" target="_blank" title="" type="button">
```

Comment:
Unlikely, because I just tried with the old 38.0.5 and it still consistently returns the correct one.
Anyway, let's move forward with this PR: undo these changes, they're preventing a passing build in Travis.